rsvg2/lib/rsvg2/handle.rb

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2017  Ruby-GNOME2 Project Team
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+module RSVG
+  class Handle
+    alias_method :initialize_raw, :initialize
+    def initialize(options={})
+      flags = options[:flags] || nil
+      file = options[:file] || nil
+      data = options[:data] || nil
+
+      if file
+        if flags
+          initialize_new_from_file(file)
+          flags = flags
```

Comment:
> initialize_new_from_gfile_sync

Where does it come from ? I did not found anything related to this in the API documentation.